### ISSUE TITLE
Uncaught TypeError: Illegal invocation

### ISSUE THREAD

**Issue by szhu** [FoldingText/foldingtext-for-atom#74]
This bug is really weird because everything was working fine yesterday and now I can't use foldingtext :(

I haven't changed my setup at all since then, and was able to reproduce the issue on a clean setup.

---

Steps to reproduce:
1. Make new user account.
2.  Download Atom and install `foldingtext-for-atom` and `foldingtext-markdown`.
3. I already have Node v0.12.7 installed via Homebrew.
4. File > New Outline.
5. This:
   <img width="1024" alt="screen shot 2015-09-18 at 1 14 00 am" src="https://cloud.githubusercontent.com/assets/1570168/9955028/90473b9c-5da2-11e5-9a6e-771bbdf1ea00.png">

**Atom Version**: 1.0.13
**System**: Mac OS X 10.10.5
**Thrown From**: [foldingtext-for-atom](https://github.com/FoldingText/foldingtext-for-atom) package, v0.9.0
### Stack Trace

Uncaught TypeError: Illegal invocation

```
At /Users/Guest/.atom/packages/foldingtext-for-atom/lib/editor/outline-editor-element.coffee:844

TypeError: Illegal invocation
  at HTMLUListElement.EventRegistery.listen.mousedown (/Users/Guest/.atom/packages/foldingtext-for-atom/lib/editor/outline-editor-element.coffee:844:68)
  at EventRegistery.handleEvent (/Users/Guest/.atom/packages/foldingtext-for-atom/lib/editor/event-registery.coffee:114:33)
  at HTMLBodyElement.<anonymous> (/Users/Guest/.atom/packages/foldingtext-for-atom/lib/editor/event-registery.coffee:3:1)

```
### Commands

```
     -0:47.9.0 settings-view:open (atom-pane.pane.active)
  3x -0:44.8.0 core:close (atom-text-editor.editor.is-focused)
     -0:28.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:04.7.0 core:close (div.settings-view.pane-item)
     -0:01.8.0 outline-editor:new-outline (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
foldingtext-for-atom, v0.9.0
foldingtext-markdown, v1.3.2

# Dev
No dev packages
```

**Comment by JohnMurga** [comment]
I am getting this on Windows today also, since I upgraded to the new Atom 1.0.13 build.
Does not happen on Atom 1.0.11.

**Comment by szhu** [comment]
Can confirm that it does not error on 1.0.11.

Still errors on Atom 1.0.15.

**Comment by jessegrosjean** [comment]
This should be fixed in latest 0.9.2 release.
